library/build.gradle

```diff
@@ -21,6 +21,14 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 22
+
+        def projects = rootProject.getProject().getChildProjects()
+        projects.each {
+            def authority = it.value.android.defaultConfig.manifestPlaceholders.downloadAuthority
```

Comment:
yeah I was thinking that as well, for ex non android modules.

Nope I can go with that, map was first solution found :)